newLib.js

```diff
@@ -0,0 +1,29 @@
+const util = require('util')
+const cp = require('child_process')
+
+const projectName = process.argv.slice(2).toString()
+
+const execAsync = util.promisify(cp.exec)
+
+async function execute(command) {
+  const logs = await execAsync(command)
+  process.stdout.write(logs.stdout)
+}
+
+const main = async () => {
+  await execute(`npx hygen bs new ${projectName}`)
```

Comment:
Wouldn't be better to have hygen and lerna in the dev dependencies? Since we are using npx here it'll fetch the package all the times from npm database?

Reply:
but are already there (devDependencies) no?